client/service/src/metrics.rs

```diff
@@ -0,0 +1,276 @@
+
+use prometheus_endpoint::{register, Gauge, U64, F64, Registry, PrometheusError, Opts, GaugeVec};
+use sysinfo::{get_current_pid, ProcessExt, System, SystemExt};
+use sc_telemetry::{telemetry, SUBSTRATE_INFO};
+use sc_client::ClientInfo;
+use crate::NetworkStatus;
+use sp_transaction_pool::PoolStatus;
+use sp_runtime::traits::{NumberFor, Block, SaturatedConversion, UniqueSaturatedInto};
+use netstat2::{TcpState, ProtocolSocketInfo, iterate_sockets_info, AddressFamilyFlags, ProtocolFlags};
+use sp_utils::metrics::GLOBAL_METRICS;
+
+struct PrometheusMetrics {
+	block_height_number: GaugeVec<U64>,
+	ready_transactions_number: Gauge<U64>,
+	memory_usage_bytes: Gauge<U64>,
+	netstat: GaugeVec<U64>,
+	load_avg: GaugeVec<F64>,
+	cpu_usage_percentage: Gauge<F64>,
+	network_per_sec_bytes: GaugeVec<U64>,
```

Comment:
Note that this is already reported by the network crate.

Reply:
It's a copy-paste from [before](https://github.com/paritytech/substrate/pull/5414/files#diff-240c17a1e4dbce4ad137b7777bf03ce4L85). If it is already reported, we might want to drop it here... which is a bit weird though, because the tick of this is currently triggered by the netstat-channel ..